pkg/discovery/stitching/stitching_manager.go

```diff
@@ -16,6 +16,8 @@ import (
 const (
 	UUID StitchingPropertyType = "UUID"
 	IP   StitchingPropertyType = "IP"
+	CNTID StitchingPropertyType = "CNT"
+	CNTFULLID StitchingPropertyType = "CNTFULL"
```

Comment:
This `StitchingPropertyType` is actually a config enum for the users to indicate if we use UUID or IP to stitch VMs.  Now we have a lot of other entities being stitched, it's probably a good idea to change it to something like `VMStitchingPropertyType`.

In other words, the two container stitching properties newly introduced here aren't really belonging to the config enum and I'd thus suggest to just simply define them as a string like the `AppStitchingAttr` below.  For example,
```
ContainerShortIdStitchingAttr string = "CNT"
ContainerFullIdStitchingAttr string = "CNTFULL"
```

Reply:
good point, I just turned them into strings